stack.yaml

```diff
@@ -1,10 +1,10 @@
-resolver: lts-14.27 # Last 8.6.5
+resolver: lts-17.12
```

Comment:
I love this update :)

Reply:
yeah, 8.10.4 windows version seems to be reliable enough to make it the default